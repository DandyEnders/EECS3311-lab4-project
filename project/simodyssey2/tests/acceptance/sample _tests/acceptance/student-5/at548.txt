pass
move(SW)
move(N)
liftoff
move(W)
move(NE)
move(E)
move(S)
move(NW)
move(SE)
test(7,12,12,23,31)
abort
move(SE)
move(NW)
liftoff
move(E)
test(29,10,38,28,31)
move(N)
land
move(SE)
move(SE)
move(N)
move(SE)
move(W)
move(NW)
move(SW)
move(NW)
move(E)
move(E)
wormhole
move(N)
move(NW)
move(NE)
test(29,23,33,46,51)
play
move(S)
move(NW)
move(W)
move(E)
move(S)
