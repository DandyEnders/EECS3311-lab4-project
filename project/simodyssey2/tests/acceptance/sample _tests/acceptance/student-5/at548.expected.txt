  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->pass
  state:0.1, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.2, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.3, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.4, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.5, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.6, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.7, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.8, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.9, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.10, error
  Negative on that request:no mission in progress.
->test(7,12,12,23,31)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],-,-,-
    [1,2]->[-8,*],-,-,-
    [1,3]->[1,P],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[2,P],[3,B],-,-
    [2,1]->-,-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[4,P],-,-,-
    [3,1]->[5,A],[-6,Y],-,-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[6,P],[-11,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[-2,W],-,-,-
    [4,2]->[7,A],[-7,*],-,-
    [4,3]->[8,B],-,-,-
    [4,4]->[9,B],[-4,*],-,-
    [4,5]->-,-,-,-
    [5,1]->[10,B],[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],-,-,-
    [5,4]->-,-,-,-
    [5,5]->[11,P],[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [3,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [5,A]->turns_left:1
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [7,A]->turns_left:2
    [8,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [9,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [10,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PB--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   A*--   B---   B*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   ----   PW--   
->abort
  state:1.1, ok
  Mission aborted. Try test(3,5,7,15,30)
->move(SE)
  state:1.2, error
  Negative on that request:no mission in progress.
->move(NW)
  state:1.3, error
  Negative on that request:no mission in progress.
->liftoff
  state:1.4, error
  Negative on that request:no mission in progress.
->move(E)
  state:1.5, error
  Negative on that request:no mission in progress.
->test(29,10,38,28,31)
  state:1.6, error
  Thresholds should be non-decreasing order.
->move(N)
  state:1.7, error
  Negative on that request:no mission in progress.
->land
  state:1.8, error
  Negative on that request:no mission in progress.
->move(SE)
  state:1.9, error
  Negative on that request:no mission in progress.
->move(SE)
  state:1.10, error
  Negative on that request:no mission in progress.
->move(N)
  state:1.11, error
  Negative on that request:no mission in progress.
->move(SE)
  state:1.12, error
  Negative on that request:no mission in progress.
->move(W)
  state:1.13, error
  Negative on that request:no mission in progress.
->move(NW)
  state:1.14, error
  Negative on that request:no mission in progress.
->move(SW)
  state:1.15, error
  Negative on that request:no mission in progress.
->move(NW)
  state:1.16, error
  Negative on that request:no mission in progress.
->move(E)
  state:1.17, error
  Negative on that request:no mission in progress.
->move(E)
  state:1.18, error
  Negative on that request:no mission in progress.
->wormhole
  state:1.19, error
  Negative on that request:no mission in progress.
->move(N)
  state:1.20, error
  Negative on that request:no mission in progress.
->move(NW)
  state:1.21, error
  Negative on that request:no mission in progress.
->move(NE)
  state:1.22, error
  Negative on that request:no mission in progress.
->test(29,23,33,46,51)
  state:1.23, error
  Thresholds should be non-decreasing order.
->play
  state:2.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   PPA*   BP--   M---   Y---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   PY--   ----   A---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    B---   ----   O---   PY--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   B---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PY--   P---   *---   PW--   
->move(S)
  state:3.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,1]
    [3,A]:[1,2,3]->[5,3,2]
    [4,B]:[1,3,1]->[5,3,3]
    [6,M]:[1,4,1]->[2,4,1]
    [8,A]:[2,5,1]->[3,1,2]
      destroyed [9,B] at [3,1,1]
    [14,P]:[5,3,1]->[5,2,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   PP-*   -P--   ----   Y---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    E---   *---   PY--   M---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -A--   ----   O---   PY--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   B---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PYP-   -AB-   *---   PW--   
->move(NW)
  state:4.0, mode:play, ok
  Movement:
    [0,E]:[2,1,1]->[1,5,2]
    [8,A]:[3,1,2]->[2,1,1]
    [12,B]:[4,4,1]->[5,5,3]
    [15,P]:[5,5,1]->[4,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   PP-*   -P--   ----   YE--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    A---   *---   PY--   M---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   ----   O---   PY--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   ----   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PYP-   -AB-   *---   -WB-   
->move(W)
  state:5.0, mode:play, ok
  Movement:
    [0,E]:[1,5,2]->[1,4,1]
    [3,A]:[5,3,2]->[4,3,2]
    [4,B]:[5,3,3]->[1,2,3]
    [5,P]:[1,3,2]->[5,4,2]
    [8,A]:[2,1,1]->[3,2,1]
    [11,P]:[3,5,1]->[2,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   PPB*   ----   E---   Y---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   PY--   M---   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   A---   O---   PY--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   WA--   ----   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PYP-   ----   *P--   -WB-   
->move(E)
  state:6.0, mode:play, ok
  Movement:
    [0,E]:[1,4,1]->[1,5,2]
    [6,M]:[2,4,1]->[3,4,3]
      reproduced [16,M] at [3,4,4]
    [11,P]:[2,5,1]->[1,5,3]
    [12,B]:[5,5,3]->[2,2,2]
      reproduced [17,B] at [2,2,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   PPB*   ----   ----   YEP-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *BB-   PY--   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   A---   O---   PYMM   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   WA--   ----   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PYP-   ----   *P--   -W--   
->move(S)
  state:7.0, mode:play, ok
  Movement:
    [0,E]:[1,5,2]->[2,5,1]
    [3,A]:[4,3,2]
    [8,A]:[3,2,1]->[4,1,1]
    [12,B]:[2,2,2]->[2,3,3]
    [15,P]:[4,5,1]->[3,5,1]
    [17,B]:[2,2,3]->[3,1,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   PPB*   ----   ----   Y-P-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   PYB-   ----   E---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    B---   ----   O---   PYMM   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    A---   ----   WA--   ----   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PYP-   ----   *P--   -W--   
